modules/ROOT/pages/view-clear-logs-task.adoc

```diff
@@ -1,26 +1,76 @@
-= View Logs
+= View the Log for a Mule App Project in Design Center
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Design Center, expand the area at the bottom of the flow design.
```

Comment:
@rddheath can we say 'In Flow Designer, click on the "Logs" button found in the bottom right pane of the page'? or something like that :)

Reply:
Your comment is about text that no longer exists in the file. The pink text is deleted text. The green text is new text.